e2e-tests/test_nessus.py

```diff
@@ -10,11 +11,16 @@ def test_nessus(self):
         """Test rapidast find expected number of findings in VAPI"""
         self.create_from_yaml(f"{self.tempdir}/nessus-deployment.yaml")
         self.create_from_yaml(f"{self.tempdir}/nessus-service.yaml")
-        wait_until_ready(label_selector="app=nessus", timeout=300)  # nessus is slow to pull and start
+        assert wait_until_ready(label_selector="app=nessus", timeout=360)  # nessus is slow to pull and start
 
         self.create_from_yaml(f"{self.tempdir}/rapidast-nessus-configmap.yaml")
         self.create_from_yaml(f"{self.tempdir}/rapidast-nessus-pod.yaml")
-        wait_until_ready(field_selector="metadata.name=rapidast-nessus")
+        # @FIX: We don't assert the container's successful completion because it
```

Comment:
:O Does this mean the nessus test should have been failing for some time already?

Reply:
Well, this test doesn’t assert much, it only retrieves logs from the RapiDAST container. While the container does generate logs, it ends with an error. It would be better to update the test to at least assert that the container completes successfully, I should have a simple solution, just remove the 'authentication' part of the config file